pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -489,8 +489,8 @@ func (c *Reconciler) reconcile(ctx context.Context, pr *v1.PipelineRun, getPipel
 	if err := resources.ValidateRequiredParametersProvided(&pipelineSpec.Params, &pr.Spec.Params); err != nil {
 		// This Run has failed, so we need to mark it as failed and stop reconciling it
 		pr.Status.MarkFailed(v1.PipelineRunReasonParameterMissing.String(),
-			"PipelineRun %s parameters is missing some parameters required by Pipeline %s's parameters: %s",
-			pr.Namespace, pr.Name, err)
+			"PipelineRun %s/%s is missing some parameters required by Pipeline %s: %s",
```

Comment:
The other flows indicate a namespace for the Pipeline as well, e.g.  on line 502:
```PipelineRun %s/%s parameters have mismatching types with Pipeline %s/%s's parameters```

The namespace in those cases is just taken from the PipelineRun though - Not sure if it should be standardized to include namespace or not

Reply:
Thanks! Have addressed :+1: